package.json

```diff
@@ -84,7 +84,6 @@
     "classnames": "2.3.1",
     "cozy-bar": "^16.0.1",
     "cozy-client": "^52.1.0",
-    "cozy-client-js": "0.20.0",
```

Comment:
It is not enough for the `cozy-client-js` removal. They are `cozy.client` global variable uses in the repo.